pkg/activator/config/store.go

```diff
@@ -36,57 +36,38 @@ func FromContext(ctx context.Context) *Config {
 	return ctx.Value(cfgKey{}).(*Config)
 }
 
-func toContext(ctx context.Context, c *Config) context.Context {
-	return context.WithValue(ctx, cfgKey{}, c)
-}
-
 // Store loads/unloads our untyped configuration.
 // +k8s:deepcopy-gen=false
 type Store struct {
 	*configmap.UntypedStore
+
+	// current is the current Config.
+	current atomic.Value
 }
 
 // NewStore creates a new configuration Store.
 func NewStore(logger configmap.Logger, onAfterStore ...func(name string, value interface{})) *Store {
-	return &Store{
-		UntypedStore: configmap.NewUntypedStore(
-			"activator",
-			logger,
-			configmap.Constructors{
-				tracingconfig.ConfigName: tracingconfig.NewTracingConfigFromConfigMap,
-			},
-			onAfterStore...,
-		),
-	}
+	s := &Store{}
+
+	// Append an update function to run after a ConfigMap has updated to update the
+	// the current state of the Config.
```

Comment:
optical illusion detected!

```suggestion
	// Append an update function to run after a ConfigMap has updated to update
	// the current state of the Config.
```

Reply:
Done, thanks!